docs/overview/shareable.rst

```diff
@@ -4,16 +4,16 @@
 Shareable
 ---------
 
-Root directory with installed packages can be shared between several local
-projects. In case if one local project trigger installing of some new third
-party package then root directory will be locked for modifications until install
-finished or interrupted. Other local project that will try to run
-``hunter_add_package`` in same root will be automatically paused and spin
-waiting for unlock. Note that different root directories do different locks,
-e.g.  triggering build of ``OpenSSL`` for ``iOS`` will not pause building of
-``GTest`` for ``Android``.  Internally this is done by ``file(LOCK ...)`` CMake
+A root directory with installed packages can be shared between several local
+projects. If one local project triggers installation of a new third
+party package, then the root directory will be locked against modifications until the install
+has either finished or cancelled. Other local projects that try to run
```

Comment:
Spell check doesn't like the word `cancelled`, see build error. You can add `cancelled` to `docs/spelling.txt` to suppress this warning because it seems both `canceled` and `cancelled` are correct (British vs US).

Reply:
Also usually build cancelled by Ctrl+C and the word `interrupted` is more accurate. For example there is no single "cancel" word in documents:
* https://en.wikipedia.org/wiki/Control-C
* http://man7.org/linux/man-pages/man7/signal.7.html

Reply:
My worry with interrupted was that it could also occur if something crashed or was killed (not sure if Hunter would leave a lock then).  I added Ctrl-C as an example of the interruption that was being referred to. Seem ok?

Reply:
> not sure if Hunter would leave a lock then

`file(LOCK ...)` command used for this and everything should be fine, OS is responsible for unlocking file if process is killed.

> I added Ctrl-C as an example of the interruption that was being referred to. Seem ok?

Ok